svf/include/SVFIR/SVFValue.h

```diff
@@ -438,22 +444,28 @@ class SVFFunction : public SVFValue
         /// Carefully! 'back' is just the last basic block of function,
         /// but not necessarily a exit basic block
         /// more refer to: https://github.com/SVF-tools/SVF/pull/1262
-        return allBBs.back();
+        return std::prev(bbGraph->end())->second;
     }
 
     inline const_iterator begin() const
     {
-        return allBBs.begin();
+        return bbGraph->begin();
     }
 
     inline const_iterator end() const
     {
-        return allBBs.end();
+        return bbGraph->end();
     }
 
-    inline const std::vector<const SVFBasicBlock*>& getBasicBlockList() const
+    inline std::vector<const SVFBasicBlock*> getBasicBlockList() const
```

Comment:
what this method for? why use transform? too complicated.

Reply:
Yes. I remove it. For all places where this API is needed, I use iterator instead.